.gitignore

```diff
@@ -24,4 +24,5 @@ yarn-debug.log*
 .yarn-integrity
 /yarn-error.log
 
-.nova
\ No newline at end of file
+.nova
+.ruby-version
```

Comment:
Not sure if this should be added. But there is no ruby-version file in the repo.